.isort.cfg

```diff
@@ -0,0 +1,7 @@
+[settings]
+profile=black
+src_paths=snuba
+multi_line_output=3
+line_length=88
+atomic=true
+include_trailing_comma=true
```

Comment:
Does profile=black not already imply most of these other settings?

Reply:
Yep, good point. I removed the unnecessary configs.